tensorflow/c/experimental/filesystem/plugins/gcs/gcs_filesystem.cc

```diff
@@ -38,8 +38,8 @@ static inline void TF_SetStatusFromGCSStatus(
 static void* plugin_memory_allocate(size_t size) { return calloc(1, size); }
 static void plugin_memory_free(void* ptr) { free(ptr); }
 
-void ParseGCSPath(absl::string_view fname, bool object_empty_ok, char** bucket,
-                  char** object, TF_Status* status) {
+void ParseGCSPath(const std::string& fname, bool object_empty_ok,
+                  std::string& bucket, std::string& object, TF_Status* status) {
```

Comment:
We should not use mutable references, according to the coding style. Mutable references make it harder to know that the argument changes on a function call.

Let's covert to pointers instead. There should not be a performance penalty https://godbolt.org/z/9w4M_6

PS: nice use of `std::move` :+1:

Reply:
Done